src/components/LearningMaterial/index.tsx

```diff
@@ -87,12 +87,18 @@ export default function LearningMaterial({
     }))
     return [
       ...getUnorderedListAccordion(
-        'Learning outcomes',
+        `${landingPageCopy?.LearningOutcomes}`,
```

Comment:
landingPageCopy doesn't need to be optional anymore, right?

Reply:
No you are right. They are required in the CMS so there is no need for optional anymore.